Sources/swift-parser-cli/Commands/PerformanceTest.swift

```diff
@@ -39,10 +40,15 @@ struct PerformanceTest: ParsableCommand {
 
     let start = Date()
     let startInstructions = getInstructionsExecuted()
-    for _ in 0..<self.iterations {
+    let parseTransition = IncrementalParseTransition()
+    var previousTreeDict: [Data:SourceFileSyntax] = [:]
+    for iter in 0..<self.iterations {
       for file in files {
+        if iter != 0 {
+          parseTransition.setupTransition(tree: previousTreeDict[file]!, edits: ConcurrentEdits(fromSequential: []))
+        }
         file.withUnsafeBytes { buf in
-          _ = Parser.parse(source: buf.bindMemory(to: UInt8.self))
+          previousTreeDict[file] = Parser.parse(source: buf.bindMemory(to: UInt8.self), parseTransition: parseTransition)
```

Comment:
I would add a flag to decide whether we want to do incremental parsing on subsequent iterations.